web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -154,6 +154,17 @@
           <span data-translate="">enableWorkflow</span>
         </a>
       </li>
+      <!--metadata workflow assistance external link(s)-->
```

Comment:
I suggest adding the following divider so that the external links are separated.

      <li role="menuitem" class="divider"
          data-ng-if="user.isConnected() && user.canEditRecord(md) && isMdWorkflowAssistEnable"></li>

Note that I don't believe these links only apply to "isMdWorkflowEnable && md.isWorkflowEnabled()" because any metadata editor may want an external link for assistant regardless of whether the system has metadata in work flow mode or not.

If the administrator decided to add these external links then they most likely want the link to appear.

Reply:
The separator is added. Also make the menu link same permission logic as this on